integration-tests/hibernate-search-orm-elasticsearch/src/test/java/io/quarkus/it/hibernate/search/orm/elasticsearch/management/HibernateSearchManagementTest.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.it.hibernate.search.orm.elasticsearch.management;
+
+import static org.hamcrest.Matchers.is;
+
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+
+@QuarkusTest
+@DisabledOnOs(OS.WINDOWS)
```

Comment:
Might I ask why? :)

Reply:
I've noticed that the tests for dev services were also disabled like that for Windows, so I've assumed that "fewer tests executed on that env is better" was the goal here 😄...

Reply:
> I've noticed that the tests for dev services were also disabled like that for Windows, so I've assumed that "fewer tests executed on that env is better" was the goal here 😄...

Hey @gsmet, could you confirm ^, please?

Reply:
To clarify, I was talking about the tests in `io.quarkus.it.hibernate.search.orm.elasticsearch.devservices` 😃

Reply:
Ok, in that case this probably doesn't make sense here.